cvat/apps/dataset_manager/bindings.py

```diff
@@ -779,6 +780,8 @@ def meta_for_task(db_task, host, label_mapping=None):
             ("start_frame", str(db_task.data.start_frame)),
             ("stop_frame", str(db_task.data.stop_frame)),
             ("frame_filter", db_task.data.frame_filter),
+            ("consensus_job_per_segment", str(db_task.consensus_job_per_segment)),
+            ("agreement_score_threshold", str(db_task.agreement_score_threshold)),
```

Comment:
What is the point in adding this information into export?

Reply:
My bad; I missed making modifications here while moving from task data to task.